src/utils/fetch.js

```diff
@@ -50,7 +52,9 @@ function convertFetchReject(url, body, error) {
  * @param result
  * @returns {Promise}
  */
-export function convertFetchSuccess(url, body, result) {
+export function convertFetchSuccess(url, body, timerId, result) {

```

Comment:
It looks like test use old signature and are failing with empty result. Please update tests.